packages/asset-swapper/src/swap_quoter.ts

```diff
@@ -36,12 +37,15 @@ export class SwapQuoter {
     public readonly expiryBufferMs: number;
     public readonly chainId: number;
     public readonly permittedOrderFeeTypes: Set<OrderPrunerPermittedFeeTypes>;
+    public readonly rfqtTakerApiKeyWhitelist: string[];
```

Comment:
Do these need to be public?

Reply:
Addressed in https://github.com/0xProject/0x-monorepo/pull/2541/commits/fa617d2e9d194b501f132129e9be08ef1d727e6f